CautionaryAlertsApi/V1/Infrastructure/IAuditable.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace CautionaryAlertsApi.V1.Infrastructure
+{
+    public interface IAuditable
```

Comment:
Is this being used?

Reply:
I thought I have removed it. Update is coming soon